app/pages/api/auth/[...nextauth].ts

```diff
@@ -150,6 +152,20 @@ export default NextAuth({
       clientSecret: process.env.COGNITO_SECRET,
       issuer: process.env.COGNITO_ISSUER,
     }),
+    Okta({
+      clientId: process.env.OKTA_ID,
+      clientSecret: process.env.OKTA_SECRET,
+      issuer: process.env.OKTA_ISSUER,
+    }),
+    SlackProvider({
+      clientId: process.env.SLACK_ID,
+      clientSecret: process.env.SLACK_SECRET,
+    }),
+    Okta({
```

Comment:
I see it being included twice. could you remove one of them?

Reply:
Sorry, let me run the code just to be sure